cmd/controller/controller.go

```diff
@@ -188,7 +188,7 @@ func (c *CmdOpts) startController(ctx context.Context) error {
 	})
 
 	if c.NodeConfig.Spec.API.ExternalAddress != "" {
```

Comment:
I think this should be also guarded by `if singleNode` check?

Reply:
@soider you marked this resolved but it's still using `externalAddress` check? Did I miss something? :)